infra/nnfw/cmake/packages/TensorFlowLiteGpu/CMakeLists.txt

```diff
@@ -0,0 +1,85 @@
+#
+# Tensorflow Lite GPU delegate library 2.4.1
+#
+
+set(TENSORFLOW_LITE_BASE ${TensorFlowGpuSource_DIR}/tensorflow/lite)
+set(TENSORFLOW_LITE_GPU_SRC_BASE "${TENSORFLOW_LITE_BASE}/delegates/gpu")
+
+set(TFLITE_GPU_CL_SRCS ${TENSORFLOW_LITE_GPU_SRC_BASE}/cl/api.cc
+                        ${TENSORFLOW_LITE_GPU_SRC_BASE}/cl/buffer.cc
+                        ${TENSORFLOW_LITE_GPU_SRC_BASE}/cl/cl_command_queue.cc
+                        ${TENSORFLOW_LITE_GPU_SRC_BASE}/cl/arguments.cc
+                        ${TENSORFLOW_LITE_GPU_SRC_BASE}/cl/cl_context.cc
+                        ${TENSORFLOW_LITE_GPU_SRC_BASE}/cl/cl_device.cc
+                        ${TENSORFLOW_LITE_GPU_SRC_BASE}/cl/cl_event.cc
+                        ${TENSORFLOW_LITE_GPU_SRC_BASE}/cl/cl_image_format.cc
+                        ${TENSORFLOW_LITE_GPU_SRC_BASE}/cl/cl_kernel.cc
```

Comment:
1. There's cmake files for TENSORFLOW_GPU in "infra/cmake/packages/TensorFlowLiteGpu/CMakeLists.txt".
    I think on of them is required.

2.  Do we need keep this cmake on infra/nnfw?

Reply:
I missed that file, 
actually we don't use that.
So I removed `TENSORFLOW_GPU` in infra/nnfw.